### DISCUSSION TITLE
Hiring rescued crew

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12403] [Ideas and suggestions]
Basically let the hiring interface sometimes have crew that cost suspiciously low for their level but the trade off is they have various negative afflictions or flaws due to them being someone that was the survivor of a submarine that wrecked. This could range fro psychosis, husk infections or any other affliction that is hidden at first. Even poisons. So you never know what you'll get.

**Comment by jigardi** [comment]
Have my upvote.
If it was possible to hire crew that YOU/YOUR CREW have previously rescued, it would be even better.
